build.gradle

```diff
@@ -91,6 +90,11 @@ bintray {
   }
 }
 
+changelog {
+  versionPlaceholder = '# Changelog'
+  changelogFilename = 'CHANGELOG.md'
+}
+
 wrapper {
   gradleVersion = '4.10.2'
   distributionType = Wrapper.DistributionType.ALL
```

Comment:
Please hook the updateChangelog task into the release plugin flow:

```
preTagCommit.dependsOn updateChangelog
```

Doing that will result in updating the changelog when `gradle release` is called

Reply:
Thanks Martin! I read about this approach in Readme, but after playing with `release` plugin locally was not sure it's going to work.